src/specs/src/board.spec.js

```diff
@@ -0,0 +1,168 @@
+import Canvanoid from "./../../Canvanoid.js";
+import Board from "./../../logic/Board.js";
+import Ball from "./../../logic/Ball.js";
+import Brick from "./../../logic/Brick.js";
+import State from "./../../logic/State.js";
+
+var expect = require('chai').expect;
+var should = require('chai').should();
+var assert = require('assert');
```

Comment:
import {expect} from 'chai'

Should do the trick, but not sure if 'chai' has something strange